<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>client_widget</class>
 <widget class="QWidget" name="client_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Online ChatRoom</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QWidget {
        background-color: #e0e0e0;
    }
    QPushButton {
        background-color: rgba(255, 255, 255, 200);
        color: #000;
        border: 1px solid #999;
        border-radius: 6px;
        font-size: 10pt;
        padding: 4px;
    }
    QPushButton:hover {
        background-color: #d0e7ff;
    }
    QLineEdit, QTextBrowser, QListWidget {
        background-color: rgba(255, 255, 255, 230);
        color: #000;
        border: 1px solid #aaa;
        border-radius: 6px;
        font-size: 10pt;
    }
    QLabel {
        color: #000;
        font-size: 10pt;
    }
   </string>
  </property>
  <widget class="QPushButton" name="msgsender_btn">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>410</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>发送消息</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>551</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="sender_edit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>551</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="filesender_btn">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>440</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>发送文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_button">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>380</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>退出登录</string>
   </property>
  </widget>
  <widget class="QListWidget" name="user_list">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>30</y>
     <width>101</width>
     <height>341</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="time">
      <property name="text">
       <string>登入时间:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="timing">
      <property name="text">
       <string>已登入:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="Now_user">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>0</y>
     <width>137</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>当前用户:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>0</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>当前在线用户列表:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exit_button</sender>
   <signal>clicked()</signal>
   <receiver>client_widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>630</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
